From 686fb3184480527d81ecc2e399b24ae676e7a1e8 Mon Sep 17 00:00:00 2001
From: Calum Lind <calumlind+deluge@gmail.com>
Date: Thu, 9 May 2013 18:53:43 +0100
Subject: [PATCH 2/5] Fix #2221 : Addtorrentdialog KeyError accessing
 download_location

Sets a default value of None for any missing config keys
---
 deluge/core/core.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deluge/core/core.py b/deluge/core/core.py
index 8a4d3f3..dd7648d 100644
--- a/deluge/core/core.py
+++ b/deluge/core/core.py
@@ -484,7 +484,7 @@ def get_config_values(self, keys):
             try:
                 config[key] = self.config[key]
             except KeyError:
-                pass
+                config[key] = None
         return config
 
     @export
-- 
1.8.3.1

